src/__tests__/index.spec.ts

```diff
@@ -382,3 +382,31 @@ describe('Constant structure checker with a margin of equivalence', () => {
     expect(hasMarginConstantStructure([1199, 1200], 2)).toBe(false);
   });
 });
+
```

Comment:
A good idea would be adding tests to cover every method of `HashMap` (but probably not necessary `HashSet`) because there are usually two main paths for `Hashable` and primitive types, so it’s good to at least be confident they don’t diverge in behavior.